From 5d79e2190fb4ad020d2c96e31a5262a8752ed7be Mon Sep 17 00:00:00 2001
From: Jianqun Xu <jay.xu@rock-chips.com>
Date: Wed, 19 Sep 2018 08:48:13 +0800
Subject: [PATCH] arm64: dts: rockchip: fix unit name without reg property
 error for rk1808

This patch will fix the warning:
Warning (unit_address_vs_reg): Node /usb@fd000000 has a unit name,
but no reg property

Change-Id: I64d306172e76b8b03772e1e40d173ba991080400
Signed-off-by: Jianqun Xu <jay.xu@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk1808.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk1808.dtsi b/arch/arm64/boot/dts/rockchip/rk1808.dtsi
index f4855a92e62f..e8ed14bdfbfc 100644
--- a/arch/arm64/boot/dts/rockchip/rk1808.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk1808.dtsi
@@ -90,7 +90,7 @@
 		#clock-cells = <0>;
 	};
 
-	usbdrd3: usb@fd000000 {
+	usbdrd3: usb {
 		compatible = "rockchip,rk1808-dwc3";
 		clocks = <&cru SCLK_USB3_OTG0_REF>, <&cru ACLK_USB3OTG>,
 			 <&cru SCLK_USB3_OTG0_SUSPEND>;
-- 
2.35.3

